recipe-generator/src/main/java/ca/derekcormier/recipe/generator/JavaGenerator.java

```diff
@@ -11,13 +14,21 @@
 public abstract class JavaGenerator extends Generator {
     public JavaGenerator(Cookbook cookbook) {
         super(cookbook);
+    }
+
+    @Override
+    protected List<Filter> getTemplateFilters() {
+        List<Filter> filters = new ArrayList<>();
```

Comment:
imo, when there's a hardcoded list of values like this, it's nicer to do `return Arrays.asList(....)`